about.html

```diff
@@ -0,0 +1,236 @@
+                    </a>
+                </div>
+                <div class="flex gap-20 items-center">
+                    <ul class="flex items-center gap-9">
+                        <li class="text-[1.2rem] text-gray-600 hover:text-orange-600 transition duration-200">
+                            <a href="./about.html">About</a>
+                        </li>
+                        <li class="text-[1.2rem] text-gray-600 hover:text-orange-600 transition duration-200">
+                            <a href="#">Program</a>
+                        </li>
+                        <li class="text-[1.2rem] text-gray-600 hover:text-orange-600 transition duration-200">
+                            <a href="#">Join</a>
+                        </li>
+                        <li class="text-[1.2rem] text-gray-600 hover:text-orange-600 transition duration-200">
+                            <a href="#">News</a>
+                        </li>
+                        <li class="text-[1.2rem] text-gray-600 hover:text-orange-600 transition duration-200">
+                            <a href="#">Sponsor</a>
+                        </li>
+                    </ul>
+
+                    <button type="button"
+                        class="text-orange-600 text-[1.4rem] border-4 hover:text-white border-orange-600 p-2 hover:bg-orange-600 transition ease-in-out duration-500 rounded-sm tracking-wider">Contact
+                        Us</button>
+                </div>
+            </div>
+            <div id="hamburger" class="cursor-pointer lg:hidden">
+                <i class="fa-solid fa-bars text-[2rem] hover:scale-110 transition duration-500"></i>
+                <!--hamburger menu-->
+            </div>
+        </nav>
+    </header>
+
+    <!--menu window-->
+    <div id="menu-window"
+        class="fixed overflow-hidden w-0 h-screen top-0 left-0 bg-zinc-700/90 backdrop-blur-lg z-50  lg:hidden transition-all ease-out duration-300">
+        <nav class="h-20 p-5 md:px-50 bg-zinc-700 text-white flex items-center drop-shadow">
+            <div class="flex w-full items-center justify-between">
+                <div class="w-14 h-auto">
+                    <a href="./index.html"><img src="./assets/img/heart.black.png" alt="home button"
+                            class="object-contain w-full h-full hover:scale-125 transition duration-500">
+                    </a>
+                </div>
+                <div id="close-btn" class="lg:hidden">
+                    <i class="fa-solid fa-times text-[2rem] hover:scale-125 transition duration-500"></i>
+                </div>
+            </div>
+        </nav>
+        <div class="flex flex-col items-center mt-24 gap-12 h-full text-center">
+            <ul class="flex flex-col items-center gap-12 text-center">
+                <li class="text-[1.6rem] text-white hover:text-orange-600 transition duration-200">
+                    <a href="./about.html">About</a>
+                </li>
```

Comment:
Kindly update this navigation link on the about page so that it points to the home page, since right now I have no way to go back to the home page 🤗